tools/mountcomposefs.c

```diff
@@ -229,14 +238,8 @@ int main(int argc, char **argv)
 
 	options.expected_fsverity_digest = opt_digest;
 
-	if (opt_verity && opt_noverity) {
-		printexit("Incompatible options verity, noverity\n");
-	}
-
-	if (opt_verity)
+	if (opt_verity || opt_digest != NULL)
```

Comment:
This is one of the bigger changes here right?  I think this makes sense...no point in verifying the digest without also doing underlying verity I guess?

Reply:
Yeah, i think it would be surprising if you successfully mounted against a particular digest, but then you can still read back modified data from the mount.